### ISSUE TITLE
Not compatible with Arduino Uno R4 boards

### ISSUE THREAD

**Issue by skylarmt** [NicoHood/HID#432]
The new Uno R4 boards have a 32-bit microcontroller (Renesas RA4M1 (Arm® Cortex®-M4)) with native USB support.

This library fails to compile, throwing the following error:

https://github.com/NicoHood/HID/blob/b16be57caef4295c6cd382a7e4c64db5073647f7/src/HID-Settings.h#L145

I spent a few hours trying to get it to work, but I kept running into various problems since I'm not totally familiar with low-level USB protocols.

**Comment by skylarmt** [comment]
I'd be happy to buy and ship @NicoHood an Uno R4 Minima to test with if he doesn't have one yet.

**Comment by rk9836** [comment]
Did you manage to resolve the issue @skylarmt?

**Comment by skylarmt** [comment]
Nope, never did.  Ended up doing something else for the project.  Would still be nice to have though...

**Comment by Michael-Z-Freeman** [comment]
> Nope, never did. Ended up doing something else for the project. Would still be nice to have though...

Hi. What did you use ?

**Comment by skylarmt** [comment]
> > Nope, never did. Ended up doing something else for the project. Would still be nice to have though...
> 
> Hi. What did you use ?

The Pi Pico with MicroPython has good USB HID support.
